docs/scripts_md/minc_deletion.md

```diff
@@ -0,0 +1,54 @@
+# NAME
+
+minc\_deletion.pl -- this script deletes files records from the database, and
+archives the actual files. Files to be deleted can be specified either based on
+the series UID or the file ID.
+
+# SYNOPSIS
+
+perl minc\_deletion.pl \`\[options\]\`
+
+Available options are:
+
+\-profile    : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-series\_uid : the series UID of the file to be deleted
+
+\-fileid     : the file ID of the file to be deleted
+
+# DESCRIPTION
+
+The program does the following:
+
+Deletes minc files from Loris by:
+  - Moving the existing files (.mnc .nii .jpg .header .raw\_byte.gz) to an
+    archive directory
+  - Deleting all related data from 2 database tables:
+    parameter\_file & files
+  - Deleting data from files\_qcstatus & feedback\_mri\_comments
+    database tables if the -delqcdata is set. In most cases
```

Comment:
add the word "option" after "-delqcdata".
Also, maybe have "-delqcdata" display as code?